### PR TITLE
initialize project with components

### PR DESCRIPTION
**Pull request by rbreva**
### initialize project with components

-Routes added.
-Project configuration.
-No styles added.

Thanks for the Review

Mauricio Mendoza

### CODE REVIEW

File: src/components/Categories.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function Categories() {
+  return (
+    <div>
+      <h2>Under construction</h2>
```
Comment (Deepakdanger) [inline src/components/Categories.js]: - [ ] [OPTIONAL] In requirement `Under construction` should be inside button.

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  render() {
+    return (
+      <BrowserRouter>
+        <div>
+          <header>
+            <h1>Bookstore CMS</h1>
+            <nav>
+              <NavLink to="/">Books</NavLink>
+              <NavLink to="/categories">Categories</NavLink>
+            </nav>
```
Comment (Deepakdanger) [inline src/App.js]: - [ ] [ OPTIONAL ] You can create a separate Navbar & import it here to display it.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
README file should be descriptive

* all main headers from the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) should be present in a student's repo
- Kindly fill all the neccesary details.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @rbreva,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**